api/proxy/common/common.go

```diff
@@ -63,14 +63,6 @@ func ParseBytesAmount(s string) (uint64, error) {
 		return uint64(num * 1024 * 1024), nil
 	case "mb":
 		return uint64(num * 1000 * 1000), nil // Decimal megabyte
```

Comment:
I think we should just remove the base 10 versions of these units... they just don't have a place in our system.

Practically, it would be too easy for someone to use `MB` and mean binary, but get decimal.

Reply:
+1